### ISSUE TITLE
tinymce is not showing fontsize fontselect fontsizeselect formatselect

### ISSUE THREAD

**Issue by X0rD3v1L** [tinymce/tinymce-react#352]
When I try to add toolbar which includes fontsize,fontselect,fontsizeselect,formatselect in my local it is not reflecting those options in the menu bar.
![issue](https://user-images.githubusercontent.com/46685302/176114524-65ca86c2-7333-477e-b29c-191b4c820b63.png)
```
<Editor
        onInit={(evt, editor) => editorRef.current = editor}
        init={{
          // forced_root_block : "<iframe>",
          height: 500,
          menubar: true,
          selector: 'textarea',
          plugins: [
            'advlist autolink lists link image charmap print preview anchor',
            'searchreplace visualblocks code fullscreen',
            'insertdatetime media table paste code help wordcount',
          ],
          toolbar: "undo redo | styleselect | fontselect | bold italic | alignleft aligncenter alignright alignjustify | outdent indent"
        }}
      />
```
Same code is working fine in react code sandbox but not in my localhost.

I am using tinymce 4.1.0 in Chrome Version 102.0.5005.115 with Windows 10 OS.

**Comment by tiny-james** [comment]
The above screenshot is TinyMCE 6. I suspect you are using different major versions on the sandbox vs the localhost.

The names for the toolbar buttons `styleselect` and `fontselect` changed in TinyMCE 6 - `styleselect` became `styles` and `fontselect` became `fontfamily`. 

Please refer to the upgrade guide:
https://www.tiny.cloud/docs/tinymce/6/migration-from-5x/

**Comment by tiny-james** [comment]
Note if you wish to stay on TinyMCE 5 set the `cloudChannel` to `"5"`.
https://www.tiny.cloud/docs/tinymce/6/react-ref/#cloudchannel

**Comment by tiny-james** [comment]
A colleague has pointed out to me that you are probably confusing the integration version with TinyMCE's version.

It's worth noting that a single version of the integration supports a large range of TinyMCE versions and it will generally try to load the latest major version available at the time of its release unless you specify the `cloudChannel`. 

For example the most recent version of the integration (4.1) should support TinyMCE 4, TinyMCE 5 or TinyMCE 6 and will load TinyMCE 6 by default. Version 3.14 of the integration supports TinyMCE 4 or TinyMCE 5 will load TinyMCE 5 by default. 

I was confused by your description because TinyMCE 4.1 is [8 years old now](https://www.tiny.cloud/docs-4x/changelog/#version410-june182014)! We may soon drop support for TinyMCE 4 in the integration as it is out of support.

**Comment by tiny-james** [comment]
Out of curiosity I tried to get TinyMCE 4.1 to load off cloud but the oldest version available was 4.4:
https://codesandbox.io/s/old-tinymce-4-4-9v2zuw?file=/src/index.js

**Comment by X0rD3v1L** [comment]
But for tiny-mce react only 4.1.0 version is available.
https://www.npmjs.com/package/@tinymce/tinymce-react

And yes now it is working fine with the new names styles and fontfamily.
